containers/centos/7/Containerfile.upstream

```diff
@@ -5,6 +5,7 @@
 #
 
 FROM quay.io/centos/centos:7
+RUN cd /etc/yum.repos.d/ && sed '/mirrorlist/d' -i *repo && sed 's|#baseurl=http://mirror.centos.org/centos/$releasever|baseurl=https://vault.centos.org/7.9.2009|' -i *repo
```

Comment:
Please, add a comment why this sequence of commands is needed, and split it over multiple lines, it improves readability. See e.g. https://github.com/teemtee/tmt/blob/76b5ff2cf67661c8d603e3f7f12eaf4ea3742f6e/containers/fedora/rawhide/Containerfile.unprivileged#L11:

```docker
    # Populate dnf cache
RUN    dnf makecache \
    # Create unprivileged user and setup sudo for it
    && dnf install -y /usr/sbin/useradd \
    && useradd fedora \
    && usermod -aG wheel fedora \
    && echo -e 'fedora\tALL=(ALL)\tNOPASSWD: ALL' >> /etc/sudoers
```

Reply:
ack, done